packages/client/src/views/Settings/ChangePhoneModal/ChangeNumberView.tsx

```diff
@@ -54,15 +60,25 @@ export function ChangeNumberView({ show, onSuccess, onClose }: IProps) {
   const restoreState = () => {
     setPhoneNumber(EMPTY_STRING)
     setIsInvalidPhoneNumber(false)
+    setUnknownError(false)
+    setShowDuplicateMobileErrorNotification(false)
   }
   const toggleDuplicateMobileErrorNotification = () => {
     setShowDuplicateMobileErrorNotification((prevValue) => !prevValue)
   }
+  const toggleUnknownErrorNotification = () => {
+    setUnknownError((prevValue) => !prevValue)
+  }
   const continueButtonHandler = async (phoneNumber: string) => {
-    const userData = await queriesForUser.fetchUserDetailsByMobile(
-      convertToMSISDN(phoneNumber, window.config.COUNTRY)
-    )
-    const mobileNumberExist = userData.data.getUserByMobile
+    const formattedNumber = convertToMSISDN(phoneNumber, window.config.COUNTRY)
+    const { data: userData, error } = await fetchUserDetailsByMobile({
+      variables: { mobile: formattedNumber }
+    })
+    if (error) {
+      setUnknownError(true)
+      return
+    }
```

Comment:
sorry, think you explained it earlier, but what was the issue using the error from Apollo?
```
  const [fetchUserDetailsByMobile, { error: unknownError }] =
    useLazyQuery<GetUserByMobileQuery>(
      GET_USER_BY_MOBILE
    )
```

Reply:
I don't think we will have any issues using the ApolloError. But since we're not going to show any specific error supposedly in the Toast, we don't need to use the ApolloError. Otherwise, it might show like this. 
![image](https://github.com/opencrvs/opencrvs-core/assets/32668488/4684cb5d-b0c0-43c9-bdd9-b1fb27d0dc8e)
Does this help?

Reply:
Ah, I'm just wondering if we could remove the 'unknownError' state variable, and show the toast if the Apollo error is present. Just to make the code a bit simpler was my thought, not necessarily to use the contents of the variable

Reply:
Ohh okay! I got it previously. But I thought that without the state, I wouldn't be able to control the `Toast` component much. I'll see if it works properly after removing the state.